test/parseclj-lex-test.el

```diff
@@ -114,15 +114,13 @@
     (should (equal (parseclj-lex-next) (parseclj-lex-token :character "\\c" 30))))
 
   (with-temp-buffer
-    (insert "\\newline\\return\\space\\tab\\a\\b\\c")
```

Comment:
Did you mean to add an extra test case instead of replacing this one? I think it'd be good if we still tested for these special cases like \newline and \return...

Reply:
It's just that this test is repeated already: https://github.com/lambdaisland/parseclj/blob/master/test/parseclj-lex-test.el#L105, I separated this change [into its own commit](https://github.com/lambdaisland/parseclj/pull/12/commits/e65eb085adede49ad0f7041368f73a0d9a0aaf6c) so it'd be easier to spot.

Reply:
got it, thanks!